--- a/toolchain/llvm_android/builders.py	2024-09-21 18:26:34.718000000 +0800
+++ b/toolchain/llvm_android/builders.py	2024-09-25 11:16:26.522196699 +0800
@@ -212,6 +212,7 @@
     def cflags(self) -> List[str]:
         cflags = super().cflags
         if self.profdata_file:
+            cflags.append('-Wno-android-unversioned-fallback')
             cflags.append('-Wno-profile-instr-out-of-date')
             cflags.append('-Wno-profile-instr-unprofiled')
         if not self.lto and self.enable_mlgo:
